modules/ROOT/pages/_partials/secrets-manager.adoc

```diff
@@ -0,0 +1,713 @@
+
+----
+> secrets-mgr:truststore:describe [flags] 
+----
+
+Returns the details of a truststore secret specified in `--id` from the secret group you pass in `--group-id`. 
+
+This command accepts the default flag `--help`.
+
+NOTE: The output does not include any sensitive or secret data.
+
+
+[[secret-truststore-list]]
+== secrets-mgr:truststore:list
+
+----
+> secrets-mgr:truststore:list [flags]
+----
+
+Lists all truststore secrets in a secret group specified in `--group-id`
+
+This command accepts the default flag `--help`.
+
+[[secret-truststore-modify]]
+== secrets-mgr:truststore:modify
+
+----
+> secrets-mgr:truststore:modify  [flags] 
+----
+
+Modifies the name or expiration date for a truststore secret specified in `--id` from the group you pass in`--groupid`. 
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--name` | New name for the truststore secret | `--name TestTruststoreSecret`
+|`--expiration-date` | New expiration date for the truststore secret | `--expiration-date 01/01/2025`
+
+|===
+
+[[secrets-truststore-replace]]
+== secrets-mgr:truststore:replace
+
+----
+> secrets-mgr:truststore:replace  [flags] 
+----
+
+Replaces an existing truststore secret you pass in `--id`, from the secret group you pass in `--group-id`, using the type you pass in `--type`. 
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--id`| Mandatory. Secret ID. | `--id 030493`
+|`--type` | Mandatory. Choose the truststore secret type. +
+Options: PEM, JKS, PKCS12, JCEKS. | `--type PEM`
+|`--truststore-file` |Mandatory. Truststore filepath. | `--truststore-file truststorefile`
+|`--algorithm` |Key manager factory algorithm for  JKS, PKCS12, and JCEKS keystore secrets. | `--algorithm SUNX509`
+|`--expiration-date`| Expiration date for the secret| `--expiration-date 01/01/2025`
+|`--name`| Name for your secret. | `--name TestSecret`
+|`--store-passphrase`|Passphrase required for JKS, PKCS12 and JCEKS keystore secrets. | `--store-passphrase examplePassphrase`
+
+|===
+
+
+[[secret-TLS-context-create]]
+== secrets-mgr:tls-context:mule:create
+
+----
+> secrets-mgr:tls-context:mule:create [flags] 
+----
+
+Creates a new Mule TLS Context secret in the secret group specified in `--group-id`, and using the name specified in `--name`.
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--group-id`| Mandatory. Secret group ID. | `--group-id 030493`
+|`--name`|Mandatory. Name for your secret. | `--name TestSecret`
+|`--tls-version` |TLS Version +
+Default: TLSv1.2. | `--tls-version TLSv1.1`
+|`--cipher` |Cipher for the specified TLS version. | `--cipher      TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256`
+|`--expiration-date`| Expiration date for the secret| `--expiration-date 01/01/2025`
```

Comment:
Please use the following example values
`--group-id` 59573b4e-cdea-4917-ac34-b047e1e94dbe
`keystore-id` 2d773060-aed0-46a7-b131-efbdb6ceff70
`trustore-id` 588c33e4-7f6f-44be-94e8-8b65a56d1670
`expiration-date` 2025-01-01